components/questions/UpdateableLatestForecast.tsx

```diff
@@ -164,10 +164,11 @@ export function UpdateableLatestForecast({
     <>
       <span
         className={clsx(
-          "mr-0.5 font-bold h-min focus-within:ring-indigo-800 ring-neutral-300 px-1 py-0.5 rounded-md shrink-0 relative flex",
+          "mr-0.5 font-bold h-min hover:ring-neutral-400 focus-within:ring-indigo-800 hover:focus-within:ring-indigo-800 ring-neutral-300 px-1 py-0.5 rounded-md shrink-0 relative flex transition-all",
```

Comment:
Looks like other commits are being included in the changes. Maybe git merge main rather than rebasing fixes this?